src/main/java/org/reaktivity/reaktor/internal/conductor/Conductor.java

```diff
@@ -111,7 +121,7 @@ public String name()
 
     private void handleCommand(
         int msgTypeId,
-        DirectBuffer buffer,
+        MutableDirectBuffer buffer,
```

Comment:
This should remain `DirectBuffer`, right?

Reply:
Yes, fixing